### PR TITLE
updating the DSC community call & resources

### PR DESCRIPTION
**Pull request by gaelcolas**
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/dsccommunity.org/189)

### CODE REVIEW

**Review by PlagueHO — CHANGES_REQUESTED** [review]
The change looks fine, but looks like the build is failing. I'll see if I can take a look tomorrow.

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/dsccommunity/dsccommunity.org/189#-:-N0yceZXAmClgnHSvsam:bwl140s)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @johlju)

File: data/en/resources.json [hunk 1]
```diff
@@ -371,115 +373,117 @@
     "PackageManagementProvider": "NuGet",
     "AdditionalMetadata": {
       "summary": "The cNtfsAccessControl module contains DSC resources for NTFS access control management.",
+      "Authors": "Serge Nikalaichyk",
+      "DscResources": "cNtfsAuditEntry cNtfsAuditInheritance cNtfsPermissionEntry cNtfsPermissionsInheritance",
+      "lastUpdated": "30/04/2022 20:36:06 -07:00",
       "CLRVersion": "4.0",
+      "created": "06/02/2019 16:41:41 -08:00",
       "NormalizedVersion": "1.4.1",
+      "updated": "2022-04-30T20:36:06Z",
```
Comment (johlju) [inline data/en/resources.json:382]: Is it the format of the `updated` property it doesn't like? But the same format was used before, but other properties are using the format `30/04/2022 20:36:06 -07:00` 🤔

File: data/en/resources.json [hunk 2]
```diff
@@ -1,84 +1,85 @@
...
-      "downloadCount": "129135",
-      "packageSize": "79650",
-      "tags": "DesiredStateConfiguration DSC DSCResourceKit DSCResource PSModule",
+      "summary": "DSC resources for installing, uninstalling and configuring Certificate Services components in Windows Server.",
+      "Authors": "DSC Community",
+      "DscResources": "AdcsAuthorityInformationAccess AdcsCertificationAuthority AdcsCertificationAuthoritySettings AdcsEnrollmentPolicyWebService AdcsOnlineResponder AdcsWebEnrollment AdscTemplate",
+      "lastUpdated": "30/04/2022 20:16:06 -07:00",
+      "created": "20/06/2020 23:17:06 -07:00",
+      "NormalizedVersion": "5.0.0",
+      "updated": "2022-04-30T20:16:06Z",
```
Comment (johlju) [inline data/en/resources.json:57]: It seems to error on this line, the first one with yesterdays date `"2022-04-30T20:16:06Z",`

**Comment by gaelcolas** [comment]
Thanks for your troubleshooting, I spotted that there were now 2 properties Updated and LastUpdated, with different date format.
I've updated the property to the Updated one which use the correct formatting for HUGO.

**Pull request merged**